app/controllers/auth_source_ldaps_controller.rb

```diff
@@ -22,8 +22,6 @@ def edit
   end
 
   def update
-    # remove from hash :account_password if blank?
-    params[:auth_source_ldap].except!(:account_password) if params[:auth_source_ldap][:account_password].blank?
```

Comment:
Now the password field is disabled in edit forms by default, so no need to do that anymore.